examples/notebooks/beam-ml/rag_usecase/beam_rag_notebook.ipynb

```diff
@@ -0,0 +1,3066 @@
+      "Inserting doc_key=doc_39_section_14, key=section_id, value=14\n",
+      "Inserting doc_key=doc_39_section_15, key=id, value=39\n",
+      "Inserting doc_key=doc_39_section_15, key=url, value=https://en.wikipedia.org/wiki/Albedo\n",
+      "Inserting doc_key=doc_39_section_15, key=title, value=Albedo\n",
+      "Inserting doc_key=doc_39_section_15, key=text, value=reflecting sunlight, but they can also serve as blankets to trap warmth. \" albedo and climate in some areas are affected by artificial clouds, such as those created by the contrails of heavy commercial airliner traffic. a study following the burning of the kuwaiti oil fields during iraqi occupation showed that temperatures under the burning oil fires were as much as colder than temperatures several miles away under clear skies. aerosol effects aerosols ( very fine particles / droplets in the atmosphere ) have both direct and indirect effects on earth's radiative balance. the direct ( albedo ) effect is generally to cool the planet ; the indirect effect ( the particles act as cloud condensation nuclei and thereby change cloud properties ) is less certain. as per spracklen et al. the effects are : aerosol direct effect. aerosols directly scatter and absorb radiation. the scattering of radiation causes atmospheric cooling, whereas absorption can cause atmospheric warming. aerosol indirect effect. aerosols modify the properties of clouds through a subset of the aerosol population called cloud condensation nuclei. increased nuclei concentrations lead to increased cloud droplet number concentrations, which in turn leads to increased cloud albedo, increased light scattering and radiative cooling ( first indirect effect ), but also\n",
+      "Inserting doc_key=doc_39_section_15, key=section_id, value=15\n"
+     ]
+    },
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "b989d9d2f49d49a2848b8aad98d633c4",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "Batches:   0%|          | 0/1 [00:00<?, ?it/s]"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "71c99fd15c904328931fc39bb635ec8a",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "Batches:   0%|          | 0/1 [00:00<?, ?it/s]"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Inserting doc_key=doc_39_section_16, key=id, value=39\n",
```

Comment:
The block of insertion outputs should be much cleaner once the print statement is removed

Reply:
@jrmccluskey I've made changes as per the code review. Please review the changes and let me know if something else needs to be changed.